src/extensions/default/QuickView/main.js

```diff
@@ -784,10 +812,15 @@ define(function (require, exports, module) {
 
     // Setup initial UI state
     setEnabled(prefs.get("enabled"), true);
+    setExtensionlessImagePreview(prefs.get("extensionlessImagePreview"));
```

Comment:
Should never write to prefs on initial load. This is unnecessary and slows down startup. I can see that this is emulating the other pref here, so remove that also if possible.